recipes/windows.rb

```diff
@@ -24,4 +24,13 @@
   version vagrant_version
   source vagrant_url
   checksum vagrant_checksum
+  returns [1641, 3010]
```

Comment:
Random OK return values from the MSI install.

Reply:
Yeah. Windows is kind of a cluster ~~when it comes to exit codes~~. It's unpredictable from one thing to the next what means `success` and what means `failure`. Worse, because the exit code value is not limited to just 1 byte (0-255), in practical terms it could be basically anything.